packages/xchain-terra/src/client.ts

```diff
@@ -179,12 +178,14 @@ class Client extends BaseXChainClient implements XChainClient {
 
   async transfer({ walletIndex = 0, asset = AssetLUNA, amount, recipient, memo }: TxParams): Promise<string> {
     if (!this.validateAddress(recipient)) throw new Error(`${recipient} is not a valid terra address`)
-    if (!isTerraAsset(asset)) throw new Error(`${assetToString(asset)} is not a valid terra chain`)
 
     const mnemonicKey = new MnemonicKey({ mnemonic: this.phrase, index: walletIndex })
     const wallet = this.lcdClient.wallet(mnemonicKey)
 
     const terraMicroDenom = getTerraMicroDenom(asset.symbol as TerraNativeAsset)
+
```

Comment:
small issue, we should do any params checks before making the  (expensive) create wallet

Reply:
Sorry, I've missed you feedback! Let's fix it in `alpha.3`

Reply:
Fix https://github.com/xchainjs/xchainjs-lib/pull/486